### PR TITLE
IST on Osmosis Frontier

### PR DESCRIPTION
**Pull request by zmanian**
Add IST to osmosis frontier

### CODE REVIEW

File: osmosis-1/osmosis-frontier.assetlist.json [hunk 1]
```diff
@@ -2874,6 +2874,32 @@
...
+          "denom": "ibc/5973C068568365FFF40DEDCF1A1CB7582B6116B731CD31A12231AE25E20B871F",
+          "exponent": 0,
+          "aliases": ["uist"]
+        },
+        {
+          "denom": "ist",
+          "exponent": 6
+        }
+      ],
+      "base": "ibc/5973C068568365FFF40DEDCF1A1CB7582B6116B731CD31A12231AE25E20B871F",
```
Comment (JeremyParish69) [inline osmosis-1/osmosis-frontier.assetlist.json:2890]: I think this needs changing. I believe SHA256(transfer/channel-320/uist) = 
92BE0717F4678905E53F4E45B2DED18BC0CB97BF1F8B6A25AFEDF3D5A879B4D5

Comment (zmanian) [reply osmosis-1/osmosis-frontier.assetlist.json]: I grabbed the wrong asset from my wallet. Thanks! Fixed

**Comment by JeremyParish69** [comment]
@zmanian Thanks. We typically like to hold off adding to this repo until after it's been added to the frontend. Do you need help with the code for the frontend repo?

**Comment by zmanian** [comment]
I think this the missing piece for IST on the front end repo.

https://github.com/osmosis-labs/osmosis-frontend/pull/941

A bunch of pieces like the logo and chain-info are already there.

**Comment by zmanian** [comment]
what else do I need to do?

**Comment by zmanian** [comment]
waiting not so patiently...

**Comment by IbrarMakaveli** [comment]
Hey @zmanian  LGTM, going to merge it

**Comment by IbrarMakaveli** [comment]
I think we should specify on the "name" key that is stablecoin

**Pull request merged**